tests/fast-integration/installer/helm.js

```diff
@@ -35,8 +35,8 @@ async function helmUninstall(release, namespace) {
   return result;
 }
 
-async function helmInstallUpgrade(release, chart, namespace, values, profile) {
-  const args = [
+async function helmInstallUpgrade(release, chart, namespace, values, profile, additionalArgs) {
```

Comment:
I'm pretty weak in JavaScript, does this have the capacity to break anything. Afair, when the parameter is not passed during function call, it will appear as `undefined` in the body and trying to check that below

Reply:
Yup, that is fine.